Plutarch.hs

```diff
@@ -22,7 +22,7 @@ module Plutarch (
   PP.PMatch,
   PO.printTerm,
   PO.printScript,
-  (PL.#$),
+  -- (PL.#$), FIXME? wtf? 
```

Comment:
elaborate?

Reply:
It somehow showed the error to be on that line which was just very weird as it's an ambiguity error that affects the whole module :)